app/locustio/jira/http_actions.py

```diff
@@ -4,10 +4,11 @@
 from locustio.jira.requests_params import Login, BrowseIssue, CreateIssue, SearchJql, ViewBoard, BrowseBoards, \
     BrowseProjects, AddComment, ViewDashboard, EditIssue, ViewProjectSummary, jira_datasets
 from locustio.common_utils import jira_measure, fetch_by_re, timestamp_int, generate_random_string, TEXT_HEADERS, \
-    ADMIN_HEADERS, NO_TOKEN_HEADERS, logger
+    ADMIN_HEADERS, NO_TOKEN_HEADERS, init_logger
 
 from util.conf import JIRA_SETTINGS
 
+logger = init_logger(app_type='jira')
 jira_dataset = jira_datasets()
```

Comment:
Need to refactor to assertion and detailed error message with content in case of a failure.
`    locust.client.post('/login.jsp', body, TEXT_HEADERS, catch_response=True)
    r = locust.client.get('/', catch_response=True)
    if not r.content:
        raise Exception('Please check server hostname in jira.yml file')
    content = r.content.decode('utf-8')`